docs/docs/develop/policies.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Secure: Dashboards"
+description: Granular access policies for dashboards 
+sidebar_label: "Secure: Dashboards"
+sidebar_position: 30
+---
+
+:::caution
+
+This is an experimental feature.
+
+:::
+
+Rill supports granular access policies for dashboards. You can define access policies for dashboards in the respective `dashboard_name.yaml` file under `policies` section.
+See policies configuration reference for all the possible options here.
+
+## Dashboard Policies
+Currently, Rill supports the following use cases for dashboards:
+- Dashboard level access - Based on this condition you can restrict access to a dashboard for specific users based on user attributes.
+- Row level filtering - Based on this condition you can restrict access to certain rows for a specific users based on user attributes.
+- Column level access - Based on this condition you can restrict access to certain columns in a dashboard for a specific users based on user attributes. 
```

Comment:
Instead of "Column level access", can we make it "Dimension/measure level access"